tar/tarsnap.1-mdoc.in

```diff
@@ -368,6 +368,9 @@ containing only the entries from
 .Ar all-backup
 containing the string
 .Sq foo .
+.It Fl -force-resources
+Force the decryption to proceed even if it is anticipated to
```

Comment:
I think this comment needs more context, to point out that we're talking about the passphrased encryption of key files...